cursorless-talon/src/spoken_forms.json

```diff
@@ -0,0 +1,260 @@
+{
+  "actions": {
+    "simple_action": {
+      "bottom": "scrollToBottom",
+      "break point": "toggleLineBreakpoint",
+      "carve": "cutToClipboard",
+      "center": "scrollToCenter",
+      "change": "clearAndSetSelection",
+      "chuck": "remove",
+      "clone up": "insertCopyBefore",
+      "clone": "insertCopyAfter",
+      "comment": "toggleLineComment",
+      "copy": "copyToClipboard",
+      "crown": "scrollToTop",
+      "dedent": "outdentLine",
+      "define": "revealDefinition",
+      "drink": "editNewLineBefore",
+      "drop": "insertEmptyLineBefore",
+      "extract": "extractVariable",
+      "float": "insertEmptyLineAfter",
+      "fold": "foldRegion",
+      "follow": "followLink",
+      "give": "deselect",
+      "highlight": "highlight",
+      "hover": "showHover",
+      "indent": "indentLine",
+      "inspect": "showDebugHover",
+      "post": "setSelectionAfter",
+      "pour": "editNewLineAfter",
+      "pre": "setSelectionBefore",
+      "puff": "insertEmptyLinesAround",
+      "quick fix": "showQuickFix",
+      "reference": "showReferences",
+      "rename": "rename",
+      "reverse": "reverseTargets",
+      "scout all": "findInWorkspace",
+      "shuffle": "randomizeTargets",
+      "snippet make": "generateSnippet",
+      "sort": "sortTargets",
+      "take": "setSelection",
+      "type deaf": "revealTypeDefinition",
+      "unfold": "unfoldRegion"
+    },
+    "callback_action": {
+      "call": "callAsFunction",
+      "scout": "findInDocument",
+      "phones": "nextHomophone"
+    },
+    "paste_action": { "paste": "pasteFromClipboard" },
+    "bring_move_action": {
+      "bring": "replaceWithTarget",
+      "move": "moveToTarget"
+    },
+    "swap_action": { "swap": "swapTargets" },
+    "wrap_action": { "wrap": "wrapWithPairedDelimiter", "repack": "rewrap" },
+    "insert_snippet_action": { "snippet": "insertSnippet" },
+    "reformat_action": { "format": "applyFormatter" }
+  },
+  "targetConnectives": {
+    "range_connective": {
+      "between": "rangeExclusive",
+      "past": "rangeInclusive",
+      "-": "rangeExcludingStart",
```

Comment:
here's an example where it would be better to map the other direction, so that we can have no default value for multiple spoken forms

Reply:
If we dont want a spoken form we could just remove the entire item?

Reply:
Then it won't show in csv